pilot/main.py

```diff
@@ -74,6 +75,11 @@ def local_print(*args, **kwargs):
     args = init()
 
     builtins.print, ipc_client_instance = get_custom_print(args)
+    if '--api-key' in args:
+        const.config.API_KEY = args['--api-key']
+
+    if '--root-folder' in args:
```

Comment:
`workspace` can now be specified as a CLI arg

https://github.com/Pythagora-io/gpt-pilot/blob/main/pilot/utils/arguments.py#L31

Reply:
Yes, but i create this branch from `ipc` branch. And `ipc` branch does not have this functionality.

Reply:
what's the `ipc` branch for?

Reply:
For ipc. We use ipc for connection beetween extension and cli

Reply:
what extensions?

Reply:
I think 've added `workspace` with the same idea in mind. An IDE plugin would be able to run GPT Pilot with `workspace=/path/to/project` and the `user_apps` table can be used to load an App config record by `workspace` and `user_id`.

Reply:
@pavel-pythagora I'd be interested to get your thoughts on https://github.com/Pythagora-io/gpt-pilot/issues/89